MAINTAINERS.md

```diff
@@ -21,13 +31,12 @@ This document explains who the maintainers are (see below), what they do in this
 | Maintainer               | GitHub ID                                                          | Affiliation |
 | ------------------------ | ------------------------------------------------------------------ | ----------- |
 | Anirudha Jadhav          | [anirudha](https://github.com/anirudha)                            |   Amazon    |
+| Daniel Doubrovkine       | [dblock](https://github.com/dblock)                                |   Amazon    |
 | Joshua Li                | [joshuali925](https://github.com/joshuali925)                      |   Amazon    |
-| Guian Gumpac             | [guiangumpac](https://github.com/guiangumpac)                      |  Bit Quill  |
-| Raymond Lum              | [raymond-lum](https://github.com/raymond-lum)                      |  Bit Quill  |
-| Yury Fridlyand           | [Yury-Fridlyand](https://github.com/Yury-Fridlyand)                |  Bit Quill  |
+| Thomas Farr              | [Xtansia](https://github.com/Xtansia)                              |   Amazon    |
+| Vacha Shah               | [VachaShah](https://github.com/VachaShah)                          |   Amazon    |
 | Max Ksyunz               | [MaxKsyunz](https://github.com/MaxKsyunz)                          |  Bit Quill  |
-| Forest Vey               | [forestmvey](https://github.com/forestmvey)                        |  Bit Quill  |
-| Mitchell Gale            | [MitchellGale-BitQuill](https://github.com/MitchellGale-BitQuill)  |  Bit Quill  |
+| Yury Fridlyand           | [Yury-Fridlyand](https://github.com/Yury-Fridlyand)                |  Bit Quill  |
 
 ## Maintainer Responsibilities
```

Comment:
Should we just link the .github MAINTAINERS like in the bottom of https://github.com/opensearch-project/anomaly-detection-dashboards-plugin/blob/8919c8f1103742e19859e34934b899fbbcb59a2b/MAINTAINERS.md to avoid duplicating?